driver/razerkbd_driver.c

```diff
@@ -1298,7 +1390,7 @@ static ssize_t razer_attr_write_set_brightness(struct device *dev, struct device
 }
 
 /**
- * Read device file "macro_mode"
+ * Read device file "set_brightness"
```

Comment:
The sysfs file is called `matrix_brightness` (if you're at it please also fix the comment at the write_set_brightness function)

Reply:
Yes, thank you